core/standard/recommendations/record-core/REC_time.adoc

```diff
@@ -0,0 +1,7 @@
+[[rec_record-core_time]]
+[width="90%",cols="2,6a"]
+|===
+^|*Recommendation {counter:rec-id}* |*/rec/core/record-contact*
+^|A |A record SHOULD include the top-level `time` member that indicates some characteristic temporal aspect of the resource that the record describes.
```

Comment:
`A record SHOULD include the top-level `time` member that indicates the temporal aspect of the resource that the record describes`

Reply:
Done.